modules/serverless-logic-data-index-graphql-queries.adoc

```diff
@@ -0,0 +1,218 @@
+// Module included in the following assemblies:
+// * serverless-logic/serverless-logic-supporting-services/serverless-logic-data-index
+
+:_mod-docs-content-type: REFERENCE
+[id="serverless-logic-data-index-graphql-queries_{context}"]
+= GraphQL queries for workflow instances and jobs
+
+The following section provides examples of GraphQL queries that allow you to retrieve data about workflow instances and jobs.
+
+[id="serverless-logic-retrieve-data-workflow-instances_{context}"]
+== Retrieve data from workflow instances
+
+You can retrieve information about a specific workflow instance by using the following query example:
+
+[source,terminal]
```

Comment:
The snippet represents a GraphQL query rather than a JSON format. There was no other option to add it in the "[source, ---]" syntax so added a `terminal`.